Entity/BlockInteractor.php

```diff
@@ -160,6 +160,9 @@ public function loadPageBlocks(PageInterface $page)
         $page->disableBlockLazyLoading();
 
         foreach ($blocks as $block) {
+            if (!$block->getEnabled()) {
```

Comment:
Maybe we should filter this in the query instead of fetching ALL blocks and filtering them afterwards